### ISSUE TITLE
Dynamic import of Astronomy

### ISSUE THREAD

**Issue by jramer** [jagi/meteor-astronomy#683]
First I'd like to say that Astronomy is great! Thanks for the hard work.

I'm trying to get Astronomy loaded dynamically. Read the closed issue #642 and thought it would be possible.

Even a new meteor project loads astronomy to the client.

My test:
```
meteor create testAst
cd testAst
meteor add jagi:astronomy
meteor --extra-packages bundle-visualizer --production
```

**Comment by lukejagodzinski** [comment]
Hmm in deed it bundles it no matter what.
I've made some tests and even with the new package it still bundles it, so probably there is no way I can leave Meteor packages out of bundle. It's probably only reserved for the NPM packages. However, if you know about any Atmosphere package that doesn't get bundled, then please let me know.

**Comment by arggh** [comment]
If I remember correctly, you could just have a bunch of `js` files inside the Meteor package, which can be imported by the app code. If they're not imported explicitly, they don't get bundled.

At least that's how I'm dynamically importing Blaze templates from a package when needed.

The crucial part is to _not_ have the files referenced in `package.js` in any of the entry points.

**Comment by lukejagodzinski** [comment]
Ok I found out that apparently you have to add `{lazy: true}` to your main
module export. I will fix that on Monday

**Comment by arggh** [comment]
Oh good! Somehow I had completely missed [that one](https://docs.meteor.com/packages/modules.html#Lazy-loading-modules-from-a-package).

**Comment by lukejagodzinski** [comment]
Fixed in 2.6.0

**Comment by jramer** [comment]
Oh! This is great! Thank you for taking the time!
Sorry I missed the conversation. (Apparently I had disabled all github notifications...)
